cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -264,3 +266,21 @@ def test_reports_delete_saved_report(appliance, request):
     view.configuration.item_select(
         item='Delete selected Saved Reports', handle_alert=True)
     assert not report.exists
+
+
+def test_reports_create_schedule_for_custom_report(
+    appliance, request, custom_report_values, schedule_data
+):
+    custom_report = appliance.collections.reports.create(**custom_report_values)
+    schedule_data["filter"] = (
+        "My Company (All Groups)",
+        "Custom",
+        custom_report.menu_name,
+    )
+    try:
+        custom_report_schedule = appliance.collections.schedules.create(**schedule_data)
+    except (MoveTargetOutOfBoundsException, NoSuchElementException):
```

Comment:
Que:
`MoveTargetOutOfBoundsException` still comming?

Reply:
I am getting the error when the test runs on PRT, but locally it gives a `NoSuchElementException`.